Database/ScanDynamo.py

```diff
@@ -0,0 +1,56 @@
+import json
```

Comment:
this does not belong in this pr, you should be creating branches to avoid this issue